manage.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+from flask.ext.script import Manager, Shell, Server
+from utils.commands import Webpack
+from server.app import create_app, db
+from server import models
+
+app = create_app()
+
+
+def _make_context():
+    return dict(app=app, db=db, models=models)
+
+manager = Manager(app)
+manager.add_command("shell", Shell(make_context=_make_context))
+manager.add_command("runserver", Server(host=app.config['HOST'], port=app.config['PORT']))
+manager.add_command("webpack", Webpack())
+
```

Comment:
Let's do this later, but I think we should have a command for compiling a dist (like the `npm run dist` from before).  I'll make a new issue for that and other deployment-related tasks.